recipes/devel/squashfs-tools.yaml

```
@@ -10,19 +10,22 @@ depends:
 checkoutSCM:
     scm: url
     url: ${GITHUB_MIRROR}/plougher/squashfs-tools/archive/${PKG_VERSION}.tar.gz
-    digestSHA256: "a7fa4845e9908523c38d4acf92f8a41fdfcd19def41bd5090d7ad767a6dc75c3"
+    digestSHA256: "277b6e7f75a4a57f72191295ae62766a10d627a4f5e5f19eadfbc861378deea7"
     stripComponents: 1
 
-buildVars: [CC, CPPFLAGS, CFLAGS, LDFLAGS]
+buildVars: [CC]
 buildScript: |
-    CFLAGS+=" -I${BOB_DEP_PATHS[utils::xz-utils-dev]}/usr/include -I${BOB_DEP_PATHS[libs::zlib-dev]}/usr/include"
-    LDFLAGS+=" -L${BOB_DEP_PATHS[utils::xz-utils-dev]}/usr/lib -L${BOB_DEP_PATHS[libs::zlib-dev]}/usr/lib"
+    mkdir build install
```

Should be `mkdir -p`. Otherwise an incremental build will fail.
```suggestion
    mkdir -p build install
```